datahub-web-react/src/app/entity/tag/TagProfile.tsx

```diff
@@ -32,6 +48,29 @@ const TitleText = styled(Typography.Title)`
     }
 `;
 
+const HeaderLayout = styled.div`
+    display: flex;
+    justify-content: space-between;
```

Comment:
Can we not style the  ant <Space> component?

Reply:
this isn't wrapping an antd component, this is just applying some flex to the contents of the header